src/hw6/check_database.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+sleep 100
+if mariadb -uroot -proot --host="mariadb" -e "USE baseball;"
```

Comment:
The logic of your if statement currently is basically "If the database exists, run hw6.sql and get the results. If not, create the database". While you do need to create the database if it doesn't exist, don't forget to run hw6.sql and get the results too. Since you have to run hw6.sql and get the results no matter what, it might be a good idea to take those commands out of the if statement and run them after.